libs/backend/src/scan/cast_vote_records/export.ts

```diff
@@ -400,5 +343,66 @@ export async function exportCastVoteRecordReportToUsbDrive({
     return exportReportResult;
   }
 
+  for (const {
+    interpretation: [sideOne, sideTwo],
+    frontNormalizedFilename: sideOneFilename,
+    backNormalizedFilename: sideTwoFilename,
+  } of resultSheetGenerator()) {
+    const canonicalizationResult = canonicalizeSheet([sideOne, sideTwo]);
+
+    if (canonicalizationResult.isErr()) {
+      return err({
+        type: 'invalid-sheet-found',
+        message: describeSheetValidationError(canonicalizationResult.err()),
+      });
+    }
```

Comment:
Should be impossible to fail canonicalization on the second pass, right?

Reply:
Should be! I think it's OK to have this redundancy though, in case the order of operations gets moved around.